packages/module/src/Battery/index.ts

```diff
@@ -1,5 +1,4 @@
-export { default } from './Battery';
-export { default as Battery } from './Battery';
```

Comment:
Make sure to test the imports in a test environment. We don't want to loose any of these references. We are only looking for the extra types reference.

Reply:
@Hyperkid123 do you think it is safer to not remove these explicit exports? I can add them back, maybe it's really better to keep them explicitly listed